src/test/java/com/amazon/ion/impl/IonReaderContinuableTopLevelBinaryTest.java

```diff
@@ -4323,4 +4360,80 @@ public void readClobValueViaGetBytes(@ConvertWith(TestUtils.HexStringToByteArray
         assertIonLobCorrectlyParsedViaGetBytes(false, IonType.CLOB, expectedBytes, inputBytes);
     }
 
+    /**
+     * Checks that the reader reads the expected annotations and values, using the provided method for retrieving the
+     * annotations.
+     */
+    private void assertAnnotationsCorrectlyParsed(
+        boolean constructFromBytes,
+        Function<String[], ExpectationProvider<IonReaderContinuableTopLevelBinary>> expectation,
+        String inputBytes
+    ) throws Exception {
+        reader = readerForIon11(hexStringToByteArray(inputBytes), constructFromBytes);
+        assertSequence(
+            next(IonType.INT), expectation.apply(new String[] {"name"}), intValue(0),
+            next(IonType.INT), expectation.apply(new String[] {"symbols", "name"}), intValue(0),
+            next(IonType.INT), expectation.apply(new String[] {"name", "symbols", "imports"}), intValue(0),
+            next(null)
+        );
+        closeAndCount();
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+        "E4 09 50 E5 0F 09 50 E6 07 09 0F 0D 50", // SIDs
+        "E7 F9 6E 61 6D 65 50 E8 0F F9 6E 61 6D 65 50 E9 1D F9 6E 61 6D 65 0F F3 69 6D 70 6F 72 74 73 50", // FlexSyms
+        "E4 12 00 50 E5 0F 24 00 00 50 E6 0E 00 09 0F 0D 50", // SIDs (multi-byte FlexUInts)
+        "E7 F2 FF 6E 61 6D 65 50 E8 3C 00 00 F9 6E 61 6D 65 50 E9 F8 00 00 00 F9 6E 61 6D 65 0F E6 FF 69 6D 70 6F 72 74 73 50", // Multi-byte FlexSyms
```

Comment:
Can you put each test case in a separate line? E.g.
```
"E4 09 50",
"E5 0F 09 50",
"E6 07 09 0F 0D 50",
```

Reply:
If I do that, I can't reuse the same assertions for all of the parameters. What I can do is use string concatenation to put individual values of each test case on separate lines.

Reply:
Oh, I didn't realize that these were all supposed to be the same data encoded in different ways.